app/team.go

```diff
@@ -618,13 +618,13 @@ func (a *App) AddUserToTeamByToken(userId string, tokenId string) (*model.Team,
 		for _, channel := range channels {
 			_, err := a.AddUserToChannel(user, channel)
 			if err != nil {
-				mlog.Error("error adding user to channel", mlog.Err(err))
+				mlog.Warn("Error adding user to channel", mlog.Err(err))
 			}
 		}
 	}
 
 	if err := a.DeleteToken(token); err != nil {
-		return nil, err
```

Comment:
This is changing the semantics of the method.  Before if token could not be deleted it returned an error (and nil team).  Is this intentional?  Are we sure it does not cause upstream issues?

Reply:
There are multiple case like this, some return error some does not. My approach here is, that main part of the function can be considered as successful here. Because the API returns `http.StatusCreated` for this flow. We had already created the resource at this point. This case is not tested as well. I can ask @jespino to be more confident about this.

Reply:
Yes, actually what you said makes sense, the user is already added to the team, the failure is on the cleanup, I think the warning is enough. If that fails (something that shouldn't happen almost ever) the problem would be that the invitation can be reused, and that problem already exists. I think the change makes sense, unless we want to make everything part of the same transaction (I think is good enough now).